src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -748,20 +753,70 @@ const UsersConfigurationRow = ({
     );
 };
 
+const SshKeysRow = ({
+    id, item, onChange, idx, removeitem,
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1150-20230821-130637-4a8374be-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)